kong/cmd/utils/log.lua

```diff
@@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
   end
 end
 
+function _M.disable()
+  old_lvl = log_lvl
```

Comment:
hence the message should be 'debug' level, so doesn't print unless your debugging, and that's when you'd want to see it.